streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaStreamingContext.scala

```diff
@@ -396,7 +396,7 @@ class JavaStreamingContext(val ssc: StreamingContext) extends Closeable {
    * Create an input stream from a queue of RDDs. In each batch,
    * it will process either one or all of the RDDs returned by the queue.
    *
-   * NOTE:
+   * @note
```

Comment:
Likewise here just want to make sure that this actually causes the following 3 lines are in the note?